No package 'libsystemd' found

Hello. Trying to install 8.2 ZTS on Catalina. Getting an error at configuration step.

### Output

```
Configuring SAPI modules
checking for Apache 2 handler module support via DSO through APXS... no
checking for setproctitle... no
checking sys/pstat.h usability... no
checking sys/pstat.h presence... no
checking for sys/pstat.h... no
checking for PS_STRINGS... no
checking for CLI build... no
checking for embedded SAPI library support... no
checking for FPM build... yes
checking for clearenv... no
checking for setproctitle... (cached) no
checking for setproctitle_fast... no
checking for library containing socket... none required
checking for library containing inet_addr... none required
checking for prctl... no
checking for procctl... no
checking for setpflags... no
checking for clock_gettime... yes
checking for ptrace... yes
checking whether ptrace works... no
checking for mach_vm_read... yes
checking if gcc supports __sync_bool_compare_and_swap... yes
checking for TCP_INFO... no
checking for TCP_CONNECTION_INFO... yes
checking for sysconf... yes
checking for times... yes
checking for kqueue... yes
checking for port framework... no
checking for /dev/poll... no
checking for epoll... no
checking for select... yes
checking for libsystemd >= 209... no
configure: error: Package requirements (libsystemd >= 209) were not met:

No package 'libsystemd' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables SYSTEMD_CFLAGS
and SYSTEMD_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
Error: Configure failed: See the pkg-config man page for more details.
```

### Command

```
phpbrew --debug install --stdout 8.2 \
    +default \
    +gd \
    +fpm \
    +debug \
    +openssl='/usr/local/opt/openssl/' \
    +bz2="$(brew --prefix bzip2)" \
    +iconv="$(brew --prefix libiconv)" \
    +zlib="$(brew --prefix zlib)" \
    -- \
    --enable-zts \
    --with-curl=/usr/local/
```

### Platform

OS:

    MacOS 10.15.7 (Catalina)

Running PHP: 

Installing PHP: 8.2

--- a/src/PhpBrew/Command/InstallCommand.php
+++ b/src/PhpBrew/Command/InstallCommand.php
@@ -439,8 +439,12 @@ class InstallCommand extends Command
                 );
             }
             if ($build->isEnabledVariant('fpm')) {
+                $addedOptions = array('--enable-fpm');
+                if (PHP_OS === 'Linux') {
+                    $addedOptions[] = '--with-fpm-systemd';
+                }
                 $sapis['fpm'] = array(
-                    'enable' => array('--enable-fpm', '--with-fpm-systemd'),
+                    'enable' => $addedOptions,
                     'disable' => array(),
                 );
             }

